io.github.kraflab.DSDA-Doom.appdata.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Ryan Krafnick -->
+<component type="desktop">
+    <id>io.github.kraflab.DSDA-Doom</id>
+    <name>DSDA-Doom</name>
+    <summary>Enhanced Doom source port</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-2.0</project_license>
+    <developer_name>Ryan Krafnick</developer_name>
+
+    <url type="homepage">https://github.com/kraflab/dsda-doom</url>
+    <url type="bugtracker">https://github.com/kraflab/dsda-doom/issues</url>
+
+    <launchable type="desktop-id">io.github.kraflab.DSDA-Doom.desktop</launchable>
+    <description>
+        <p>
+            This is a successor of prboom+ with many new features, including:
+        </p>
+
+        <ul>
+            <li>Heretic, Hexen, MBF21, Doom-in-Hexen, UDMF, and MAPINFO support</li>
+            <li>In-game console and scripting</li>
+            <li>Full controller support</li>
+            <li>Palette-based opengl renderer</li>
+            <li>Debugging features for testing</li>
+            <li>Strict mode for speedrunning</li>
+            <li>Various quality of life improvements</li>
+            <li>Advanced tools for TASing</li>
+            <li>Rewind</li>
+        </ul>
+    </description>
+
+    <screenshots>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/flathub/io.github.kraflab.DSDA-Doom/master/images/screenshot_doom.png</image>
```

Comment:
1. never reference the tip of branch as it is not immutable.
2. this repo doesn't exist anyway.

This applies to the other screenshots.

Ideally these would be upstream.